### ISSUE TITLE
[Accessibility] Keyboard Navigation: "Adaptive cards Media” button is not accessible with swipe navigation on “Media.Sources” page.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6280]
### Target Platforms

iOS

### SDK Version

App Version: 1.0 (1.3.0.20200820.7)

### Application Name

Visualizer

### Problem Description

[29527290](https://microsoft.visualstudio.com/OS/_workitems/edit/29527290)

"Adaptive cards Media” button is not accessible with swipe navigation on “Media.Sources” page.

### Card JSON

[Media.Sources.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.1/Elements/Media.Sources.json)

**Comment by jwoo-msft** [comment]
closed the issue as this is already fixed. the issue creator re-opened the issue, but the reason for re-opening concerns about a new issue that is different from the original issue.
Please check the comments in [29527290](https://microsoft.visualstudio.com/OS/_workitems/edit/29527290).

**Comment by vagpt** [comment]
Closing this issue as per the above comment
